pkgs/development/libraries/libsmartcols/default.nix

```
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, autoreconfHook, pkgconfig, python3 }:
+
+stdenv.mkDerivation rec {
+  name = "libsmartcols";
+  version = "v2.36.1";
+
+  nativeBuildInputs = [ autoreconfHook pkgconfig python3 ];
+
+  src = fetchFromGitHub {
+    owner = "karelzak";
+    repo = "util-linux";
+    rev = version;
+    sha256 = "0z7nv054pqhlihqiw0vk3h40j0cxk1yxf8zzh0ddmvk6834cnyxs";
+  };
+
+  configureFlags = ["--disable-all-programs" "--enable-libsmartcols"];
```

```suggestion
  configureFlags = [ "--disable-all-programs" "--enable-libsmartcols" ];
```

This is how it's formatted with nixpkgs-fmt. Personal preference, feel free to ignore.